zproject/prod_settings_template.py

```diff
@@ -243,6 +243,19 @@
     # "org_membership": "department",
 }
 
+# Allows more specific access to Organizations
+# This is ignored when org_membership is set
+# This contains a map with the name of the Organizations (subdomain)
+# This contains a map with a attribute and a required value.
+# If one of these value is contained in the attribute access is granted
+# If nothing matches access through LDAP is not granted (but maybe still by different backends)
```

Comment:
Let's move most of the documentation to ReadTheDocs, and have just an example here.